src/Neo/Neo.csproj

```diff
@@ -30,6 +30,7 @@
     <InternalsVisibleTo Include="Neo.SmartContract.Testing" />
     <InternalsVisibleTo Include="Neo.SmartContract.TestEngine" />
     <InternalsVisibleTo Include="Neo.Plugins.RpcServer.Tests" />
+    <InternalsVisibleTo Include="neo-cmd" />
```

Comment:
It's strange to me have two clients in the same repo, why don't improve the original one?

Reply:
It as bad as if I were to walk in outer space without a helmet on my head. What I am saying is it is to far gone, and needs to be redesigned anyways. This will solve tons of problems and issue, existing and future ones.

Reply:
A whole new CLI will lead to code bloat and confusion for users. We need to improve on the CLI we have (keeping it compatible in the process).

Reply:
Then old cli will never be fixed. or run as a service